webapp/src/modules/address/reducer.js

```diff
@@ -153,6 +161,97 @@ export function addressReducer(state = INITIAL_STATE, action) {
             }
           }
         }
+        case EDIT_ESTATE_PARCELS_SUCCESS: {
+          const { parcels, type } = transaction.payload
+          const user = state.data[transaction.from]
+          let updatedParcelIds = []
+
+          if (type === REMOVE_PARCELS) {
+            updatedParcelIds = [
+              ...user.parcel_ids,
+              ...parcels.map(p => buildCoordinate(p.x, p.y))
+            ]
+          } else {
+            updatedParcelIds = user.parcel_ids.filter(
+              parcelId =>
+                !parcels.some(
+                  estateParcel =>
+                    buildCoordinate(estateParcel.x, estateParcel.y) === parcelId
+                )
+            )
+          }
+
+          return {
+            ...state,
+            data: {
+              ...state.data,
+              [transaction.from]: {
+                ...user,
+                parcel_ids: updatedParcelIds
+              }
+            }
+          }
+        }
+        case TRANSFER_ESTATE_SUCCESS: {
```

Comment:
This is removing the estate from the old owner but it's not adding it to the new owner. Check TRANSFER_PARCEL_SUCCESS reference implementation:

```js
case TRANSFER_PARCEL_SUCCESS: {
          const { oldOwner, newOwner, x, y } = transaction.payload
          const parcelId = buildCoordinate(x, y)
          const oldOwnerAddress = state.data[oldOwner] || { ...EMPTY_ADDRESS }
          const newOwnerAddress = state.data[newOwner] || { ...EMPTY_ADDRESS }
          return {
            ...state,
            data: {
              ...state.data,
              [oldOwner]: {
                ...oldOwnerAddress,
                parcel_ids: oldOwnerAddress.parcel_ids.filter(
                  x => x !== parcelId
                )
              },
              [newOwner]: {
                ...newOwnerAddress,
                parcel_ids: newOwnerAddress.parcel_ids.concat(parcelId)
              }
            }
          }
        }
```